draft-ietf-rats-architecture.md

```diff
@@ -897,3 +899,210 @@ Thomas Hardjono created older versions of the terminology section in collaborati
+        |                                                 |
+        |                                              time(OP)
+~~~~
+
+In this example solution, the Verifier can check whether the Evidence is
+fresh at time(RG) by verifying that `time(RG) - time(VN) < Threshold`.
+
+The Verifier cannot, however, simply rely on a Nonce to
+determine whether the value of a claim is recent, since the claim value
+might have been generated long before the nonce was sent by the Verifier.
+However, if the Verifier decides that the Attester can be trusted to
+correctly provide the delta time(EG)-time(VG), then it can determine recency
+by checking `time(RG)-time(VN) + time(EG)-time(VG) < Threshold`.
+
+Similarly if, based on an Attestation Result from a Verifier it trusts,
+the Relying Party decides that the Attester can be trusted to correctly
+provide time deltas, then it can determine whether the Attestation
+Result is fresh by checking 
+`time(OP) - time(VR) + time(RR)-time(EG) < Threshold`.
+Although the Nonce2 and time(RR)-time(EG) values cannot be inside
+the Attestation Result, they might be signed by the Attester such
+that the Attestation Result vouches for the Attester's signing
+capability.
+
+The Relying Party must still be careful, however, to not allow continued 
+use beyond the period for which it deems the Attestation Result to remain
+valid.  Thus, if the Attestation Result sends a validity lifetime
+in terms of time(RX)-time(RG), then the Relying Party can check
+`time(OP) - time(VR) < time(RX)-time(RG)`.
+
+## Example 3: Timestamp-based Background-Check Model Example
+
+The following example illustrates a hypothetical Background-Check Model
+solution that uses timestamps and requires roughly synchronized
+clocks between the Attester, Verifier, and Relying Party.
+
+~~~~
+.----------.         .---------------.              .----------.
+| Attester |         | Relying Party |              | Verifier |
+'----------'         '---------------'              '----------'
+  time(VG)                   |                           |
+        |                    |                           |
+        ~                    ~                           ~
+        |                    |                           |
+  time(EG)                   |                           |
+        |----Evidence------->|                           |
+        |                 time(ER)--Evidence------------>|
+        |                    |                        time(RG)
+        |                 time(RA)<-Attestation Result---|    
+        |                    |        {time(RX)}         |
+        ~                    ~                           ~
+        |                    |                           |
+        |                 time(OP)                       |
+~~~~
+
+The time considerations in this example are equivalent to those
+discussed under Example 1 above.
+
+## Example 4: Nonce-based Background-Check Model Example
```

Comment:
Shouldn't 'Nonce' be returned to Verifier from RP?

Reply:
Yes thanks.  Fixed diagram.